quickstarts/kind/index.md

```diff
@@ -0,0 +1,112 @@
+
+Kind, on the other hand, requires no additional VM - it simply runs as a linux container with a set of processes using the same Linux kernel used by your Docker daemon.
+For this reason it is faster to start, and consumes less CPU, and RAM than the alternatives.
+
+# Installing the dependencies
+
+This quickstart assumes that you have the latest version of the `kind` binary, which you can get [here](https://github.com/kubernetes-sigs/kind/releases).
+
+Kind requires a running Docker Daemon. There are different Docker options depending on your host platform.
+You can follow the instructions [here](https://docs.docker.com/get-docker/).
+
+You'll also need `kubectl` binary, which you can get by following the instructions [here](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+
+Once you have all the binaries installed, and a Docker daemon running, make sure everything works:
+
+```shell
+# Validate docker installation
+docker ps
+docker version
+
+# Validate kind
+kind version
+
+# Validate kubectl
+kubectl version
+```
+
+# Configuring the Docker daemon
+
+If your Docker Daemon runs as a VM you'll most likely need to configure how much memory the VM should have, how many CPUs, how much disk space, and swap size.
+Make sure to assign at least 2 CPUs, and preferably 4 Gb or more of RAM. Consult Docker documentation for you platform.
+
+# Starting Kubernetes cluster
+
+This will start a local development cluster of [Kubernetes Kind](https://github.com/kubernetes-sigs/kind) which installs as a single docker container.
+
+```shell
+kind create cluster
+```
+
+# Applying Strimzi installation file
+
+Before deploying Strimzi Kafka operator, let's first create our `kafka` namespace:
+
+```shell
+kubectl create namespace kafka
+```
+
+Next we apply the Strimzi install files, including `ClusterRoles`, `ClusterRoleBindings` and some **Custom Resource Definitions** (`CRDs`). The CRDs define the schemas used for declarative management of the Kafka cluster, Kafka topics and users.
+
+```shell
+curl -L https://github.com/strimzi/strimzi-kafka-operator/releases/download/{{site.data.releases.operator[0].version}}/strimzi-cluster-operator-{{site.data.releases.operator[0].version}}.yaml \
+  | sed 's/namespace: .*/namespace: kafka/' \
+  | kubectl apply -f - -n kafka 
+```
+
+Note how we set all the `namespace` references in downloaded .yaml file to `kafka`. By default they are set to `myproject`.
+But we want them all to be `kafka` because we decided to install the operator into `kafka` namespace, which we achieve by specifying `-n kafka` when running `kubectl apply` ensuring that all the definitions and the configurations are installed in `kafka` namespace rather than the `default` namespace.
+
+If there is a mismatch between namespaces, then the Strimzi Cluster Operator will not have the necessary permissions to perform its operations. 
```

Comment:
Maybe we should mention the error the user would see, since we regular have to help people on this point on Slack.

Reply:
What if we create Troubleshooting.md document where we collect all sorts of troubles, and add a link here to point to the exact problem we have in mind?

Reply:
There is an [existing document](https://github.com/strimzi/strimzi-kafka-operator/blob/master/documentation/modules/appendix_faq.adoc) that seems a fitting place for this. Or not? That document is rather hard to read ...